demosplan/DemosPlanCoreBundle/Addon/Composer/PackageInformation.php

```diff
@@ -41,13 +41,17 @@ public function reloadPackages(): void
 
         $packageListPath = include_once $installedPackagesPath;
 
+        if (true === $packageListPath) {
+            return;
+        }
+
         if (!array_key_exists('versions', $packageListPath)) {
             return;
         }
 
         $this->addonPackages = array_filter(
             $packageListPath['versions'],
-            static fn ($version) => AddonRegistry::ADDON_COMPOSER_TYPE === strtolower($version['type'])
+            static fn ($version) => AddonRegistry::ADDON_COMPOSER_TYPE === strtolower($version['type'] ?? '')
```

Comment:
Since this only affects the actual install step of an addon, I agree with @graupnerdemos here, that it should fail louder. Proposal for that in [e71b499](https://github.com/demos-europe/demosplan-core/pull/494/commits/e71b4993ea0986235b2ff7b602616052bca5fb7e)